recipes/cusparselt/meta.yaml

```
@@ -0,0 +1,96 @@
+  # url: https://developer.download.nvidia.com/compute/libcusparse_lt/{{ version }}/local_installers/libcusparse_lt_{{ version }}.exe  # [win64 and cuda_compiler_version in ("10.1", "10.2", "11.0")]
+
+
+  # TBD
+  sha256: 63a5a854b2df4477ae48284a23c1b7c7632c23fe418434fa70a2ff8ac238b27e  # [linux64]
+  # sha256: d914a721b8a6bbfbf4f2bdea3bb51775e5df39abc383d415b3b06bbde2a47e6e  # [ppc64le]
+  # sha256: 031b75946d2ce22f8c2736f7b4216b9377406afcb35012093a66e2801bfe2990  # [aarch64]
+  # sha256: 88d3e2c662e601214ef5caa26ca2db0492eab6ec3bf9dc9a746bb1a7b8c3aab2  # [win64]
+
+build:
+  number: 0
+  # skip: True  # [win32 or cuda_compiler_version in ("None", "9.2", "10.0")]
+  skip: True  # [not linux or cuda_compiler_version != "11.0"]
+  script:
+    - mkdir -p $PREFIX/include                                                # [unix]
+    - cp -r include/* $PREFIX/include/                                        # [unix]
+    - mkdir -p $PREFIX/lib64                                                  # [unix]
+    - mv lib64/{{ cuda_compiler_version }}/libcusparseLt.so* $PREFIX/lib64/   # [unix]
+
+    # - mkdir %LIBRARY_INC%                                                                                                 # [win64]
+    # - copy %SRC_DIR%\\libcusparse_lt\\libcusparse_lt\\include\\cusparseLt.h %LIBRARY_INC%\\                               # [win64]
+    # - mkdir %LIBRARY_INC%\\cusparseLt                                                                                     # [win64]
+    # - copy %SRC_DIR%\\libcusparse_lt\\libcusparse_lt\\include\\cusparseLt\\types.h %LIBRARY_INC%\\cusparseLt              # [win64]
+    # - mkdir %LIBRARY_BIN%                                                                                                 # [win64]
+    # - copy %SRC_DIR%\\libcusparse_lt\\libcusparse_lt\\lib64\\{{ cuda_compiler_version }}\\cusparseLt.dll %LIBRARY_BIN%\\  # [win64]
+    # - mkdir %LIBRARY_LIB%                                                                                                 # [win64]
+    # - copy %SRC_DIR%\\libcusparse_lt\\libcusparse_lt\\lib64\\{{ cuda_compiler_version }}\\cusparseLt.lib %LIBRARY_LIB%\\  # [win64]
+  ignore_run_exports_from:
+    - {{ compiler('c') }}    # [linux]
+    - {{ compiler('cuda') }}  # [cuda_compiler_version == "11.0"]
+  run_exports:
+    - {{ pin_subpackage('cusparseLt', exact=True) }}
+  missing_dso_whitelist:
+    - '*/libgomp.*'  # [linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cuda') }}
+    - sysroot_linux-64 2.17  # [linux64]
+    - libgomp  # [linux]
+  run:
+    # Only GCC_3.0 or older symbols present
+    - libgcc-ng >=3.0     # [linux]
+    # Only GLIBCXX_3.4 or older symbols present
+    - libstdcxx-ng >=3.4  # [linux]
+    - cudatoolkit 11.*                         # [cuda_compiler_version not in ("10.1", "10.2")]
```

This only builds on CUDA 11.0 based on the `skip` above. So think we can just drop this selector

```suggestion
    - cudatoolkit 11.*
```

---

Done